### ISSUE TITLE
UnstaisfiedLinkError: no lpsolve55 in java.libary.path

### ISSUE THREAD

**Issue by den1den** [rapidprom/rapidprom-source#44]
After importing an external process (https://www.dropbox.com/s/w598mgvlcfy7lvi/UnstisfiedLinkError.zip?dl=1) to a empty project, and setting up the right directories in the macro's and loops i get to following error when trying to execute the 'Conformance Checking' block:

Exception: java.lang.NoClassDefFoundError
Message: Could not initialize class lpsolve.LpSolve

(full stacktrace is in the zip)
I'm using RapidMiner Studio 7.3.001 rev: c1afdd platform: WIN64
with RapidProM 4.0.1
on W10Pro 64bit version 1607 with Java 1.8.0_51 (64 bit)

Reinstallation of RapidMiner did not fix it
Removing my ~\.RapidMiner folder and reinstalling RapidProm also did not fix it
Rebooting my PC also did not fix it

**Comment by den1den** [comment]
Most simplest form with just a 'Conformance Check' also fails:
https://www.dropbox.com/s/20x5kebzn7cq3cy/NoClassDefFoundError.zip?dl=0

**Comment by abolt** [comment]
I was able to reproduce this error.
I am currently looking into it.

**Comment by abolt** [comment]
Hi Dennis,

I noticed that the Petri net was generated using the "event name + lifecycle" classifier (the transition names end with a "+")
However, the event log does not define such classifier, but the "Event name".

Please discover the model and then do conformance checking using the same classifier. 
I have tested this solution on my computer and it works.

Please let me know if it works for you as well.

Cheers,

Alfredo.

**Comment by den1den** [comment]
I double check everything but each time when i start the process it switches back.

The input for the Conformance checking has an "(Event Name AND Lifecycle transition)"-classifier, as seen below.
![image](https://cloud.githubusercontent.com/assets/856627/21468089/8afc6b52-ca03-11e6-8197-94f90851aa0e.png)

It is also possible to select this on the 'Conformance Checking' module. But as soon as i start the process this option dissapears and only the classifier 'Event Name' remains. Then the following error is thrown again:
Exception: java.lang.NoClassDefFoundError
Message: Could not initialize class lpsolve.LpSolve

I've now tried many combinations of clasifiers but each time when I run the process the Conformance Checking module switches to a clasifier that doesn't exist.

**Comment by abolt** [comment]
Hi Dennis,

I see two ways to fix it.

1) Discover the petri net using the "Event name" classifier, and do conformance checking using the same classifier.
2) use the "Add Classifier" operator before the conformance ckecking. This operator gets rid of all previous classifiers and sets the one you want, which in this case is "Event name AND lifecycle transition"

Let me know how it goes.

Cheers,

**Comment by den1den** [comment]
The silent change of classifier in the conformance module does not happen anymore but it still crashed. Reinstalling everything did not change anything. Also all clasifiers that are used for classification seem to be present in the logs. I tried both proposals:

1. I tried to export the splitted log with a classifier 'Event Name'. And also mined with an 'Event Name' classifier. I still get the same error.
2. I double checked that such a 'Event name AND lifecycle transition' clasifier was present before the 'Inductive Miner' and 'Conformance Checking'. The operator does not get rid of any previous clasifiers, it just adds one, as seen below. 
The 'Conformance Checking' will not change clasifier anymore after running, so that seems ok. But the same error presists when executing the program (with or without a double clasifier). It also does not give any more details then the stacktrace so i'm also not sure what it's looking for.
![image](https://cloud.githubusercontent.com/assets/856627/21472113/a50301ba-caca-11e6-93fc-7c3fc8e94415.png)

I include all intermediate steps for the last run once more here: https://www.dropbox.com/s/g9qdevd57upztxi/complete.zip?dl=0
So that project does complete at your installation?
